Add contribution and reviewer checklist to issue (#1145)
vmware-tanzu/tanzu-dev-portal: .github/ISSUE_TEMPLATE/content-proposal.md

- Expertise in the particular topic to be discussed

## Level of content
Level of content as it pertains to the topic proposed. Delete those not applicable.
- 101: "Awareness / Exploration"
- 201: "Prototyping / Deploying"
- 301: "Scaling
- Content Series: To include multiple levels of content (Note: Series pieces should either be able to be consumed independently, or very explicit in the title, and with links in the body of the content that they are part of a series)

## Contributor Checklist
1. [x] I have added a card to the [GitHub Content Backlog](https://github.com/vmware-tanzu/tanzu-dev-portal/projects/7) and/or moved it to the proper column. See [Content Prewriting and Ideation](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Prewriting-Ideation-(GitHub-and-Google-Docs-Method))

> 2. [ ] **If contributing via Google Docs**: I have added a link to my document to this issue, and made is visible with comment rights to everyone.

Reviewer: ```suggestion
2. [ ] **If contributing via Google Docs**: I have added a link to my document to this issue, and set the document's Share settings to "Anyone on the internet with this link can comment".
```

Reviewer: ![Screen_Shot_2021-08-04_at_3_19_22_PM](https://user-images.githubusercontent.com/13995/128256780-a6747cf7-e2d8-4c79-ab8d-ebb861e2d8e2.png)

Consider referencing this or similar screenshot.

4. [ ] The front matter (metadata) is filled out completely (e.g. description, topics, tags, author, etc - as appropriate for the content type). See [Drafting Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Drafting-Content-(GitHub-Method))
5. [ ] My content has been reviewed by my team for technical accuracy. See [Editing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Editing-Content-(GitHub-and-Google-Docs-Method))
6. [ ] My content has beed reviewed and edited by a copy editor. See [Editing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Editing-Content-(GitHub-and-Google-Docs-Method))
7. [ ] **If applicable**: My content has gone through a legal review (**VMware Employees Only**). See [Editing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Editing-Content-(GitHub-and-Google-Docs-Method))

> 8. [ ] I have tested my content locally and everything passes. See [Testing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Testing-Content-(GitHub-Method))

Reviewer: ```suggestion
8. [ ] **If applicable**: I have tested my content locally and everything passes. See [Testing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Testing-Content-(GitHub-Method))
```

Reviewer: As not all content is "testable".

9. [ ] **Optional**: I have filled out the [Social Media request form](https://docs.google.com/forms/d/e/1FAIpQLSfyWYEL6XvDRfMRW1GBfYKV4Aq3Li2EzBz0HY8fE37Y2tSoRw/viewform) (optional) (**VMware Employees Only**). See [Next Steps](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Next-Steps)
10. [ ] **If applicable**: I have created an author page and opened a pull request to get it added. See [Next Steps](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Next-Steps)

## Reviewer Checklist

> * [ ] Review if the content is a good fit for the Tanzu Developer Center. 

Reviewer: ```suggestion
* [ ] Review if the content is a good fit for the Tanzu Developer Center. See [Content Prewriting and Ideation](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Prewriting-Ideation-(GitHub-and-Google-Docs-Method))
```

* [ ] There this GitHub issue has been added to the [content backlog project](https://github.com/vmware-tanzu/tanzu-dev-portal/projects/7) 
* [ ] Ensure with the author that the content has been reviewed for technical accuracy.
* [ ] Ensure with the author that the content has been through copy editing.

> * [ ] I**If applicable**: Ensure with the author that the content has been through Legal review.

Reviewer: ```suggestion
* [ ] **If applicable**: Ensure with the author that the content has been through Legal review.
```
